rfd/0081/README.md

```diff
@@ -4,3 +4,58 @@ state: predraft
 ---
 
 # RFD 81 servicesHealth implementation
+
+The Mariposa (RFD36) health check agent will be responsible for monitoring the health of user services. The Health Check agent process(es) will read user defined health check definitions from the Project service, and coordinate which health check process will be responsible for running any given definition. The Project service is read at agent start time, then polled on an interval. There's an endpoint exposed by the agent that will trigger a poll as well.
```

Comment:
This rewrite works well.

> The Project service is read at agent start time, then polled on an interval.

You [correctly pointed out an error I made in an earlier comment](https://github.com/misterbisson/rfd/pull/2#discussion_r97640324) when I suggested the Healthcheck agent should poll the Convergence service. 

The thing I had in mind in suggesting that was the the Convergence service might be responsible for starting the Healthcheck service and reprovisioning it if it were to fail (because the CN its on is down, for example). In that case, regularly polling the Convergence service as a heartbeat makes sense.

I'm not committed to that architecture, so perhaps I'm just trying to explain my previous comment.

In describing that, however, I've come to a further question: should we preemptively turn off health checks before stopping an instance for a scale down or any other reason? That seems to be the domain of the convergence service, but it also might be needlessly more complex than simply ignoring failed health checks that might be reported in the race.